code-of-conduct.md

```diff
@@ -0,0 +1,126 @@
+  asked
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate
+
+The goal of the standards and moderation guidelines outlined here is to build
+and maintain a respectful community. We ask that you don’t just aim to be
+"technically unimpeachable", but rather try to be your best self. 
+
+We value many things beyond technical expertise, including collaboration and
+supporting others within our community. Providing a positive experience for
+other community members can have a much more significant impact than simply
+providing the correct answer.
+
+## Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable
+behavior and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or
+reject messages, comments, commits, code, issues, and other contributions
+that are not aligned to this Code of Conduct, or to ban temporarily or
+permanently any community member for other behaviors that they deem
+inappropriate, threatening, offensive, or harmful.
+
+## Moderation
+
+Instances of behaviors that violate the Adafruit CircuitPython Code of Conduct
+may be reported by any member of the community. Community members are
+encouraged to report these situations, including situations they witness
+involving other community members.
+
+You may report in the following ways:
+
+In any situation, you may send an email to <support@adafruit.com>.
+
+On the Adafruit Discord, you may send an open message from any channel
+to all Community Helpers by tagging @community helpers. You may also send an
+open message from any channel or a direct message to @kattni#1507, 
+@tannewt#4653, @Dan Halbert#1614, @cater#2442, @sommersoft#0222, or @Andon#8175.
+
+In situations on Discord where the issue is particularly egregious, possibly
+illegal, requires immediate action or violates the Discord terms of service,
+you should also report the message directly to Discord.
+
+These are the steps for upholding our community’s standards of conduct.
+
+1. Any member of the community may report any situation that violates the
+Adafruit CircuitPython Code of Conduct. All reports will be reviewed and 
+investigated, and will result in a response that is deemed necessary and 
+appropriate to the circumstances.
+2. Moderators will first respond to such behavior with a warning.
+3. We have a soft "three strikes" policy - we are willing to give a user the
+chance to change their behavior. However, how the user respond to the
+warning is important - it will determine whether or not the user receives
+further chances.
```

Comment:
I think this can be folded into 4 instead of being on its own by adding a caveat for repeated warnings. 2 is "warn the user", 4 and 5 (would be 3 and 4) are how the person responds.